src/nulog-inference-service/requirements.txt

```diff
@@ -0,0 +1,13 @@
+scipy
+numpy==1.17.4
+pandas==0.25.3
+deap==1.3.1
+recommonmark==0.6.0
+scikit_learn==0.22.2.post1
+pika
+torchvision==0.7.0
+torch==1.6.0
+boto3
+asyncio-nats-client
+elasticsearch
+aiohttp
```

Comment:
Pls add explicit versions to all packages

Reply:
fixed.